src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/_print.scss

```diff
@@ -0,0 +1,26 @@
+@media screen {
+    .print-only {
+        display: none;
+    }
+
+    .screen-only {
+        display: block;
+    }
+}
+
+@media print {
+    .print-only {
+        display: block;
+    }
+
+    .screen-only {
+        display: none;
+    }
```

Comment:
I only use the utility classes in the case we need a special print header (which we do) in which case I think it's valid.